Cargo.toml

```diff
@@ -43,6 +43,9 @@ time = "0.3.7"
 tokio = { version = "1.17", features = ["full"] }
 tower = "0.4"
 
+minitrace = "0.4.0"
+minitrace-jaeger = "0.4.0"
```

Comment:
opendal is a library and we should not depend on `minitrace-jaeger`.

Reply:
`// report trace
let socket = SocketAddr::new("127.0.0.1".parse().unwrap(), 6831);
minitrace_jaeger::report_blocking(socket, &bytes).expect("report error");`

minitrace_jaeger is a collector to collect span and send them to jaeger ui so we can see that 
![image](https://user-images.githubusercontent.com/100247023/159598969-e138c566-853b-437b-acd9-b1ea97f63134.png)

Reply:
Applications/binaries should have a collector like `minitrace_jaeger` so that they can collect the trace that opendal report. But opendal should not provide a collector, or we could conflict with the one provided by the app.